libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Util/SFSDKAnalyticsLogger.h

```diff
@@ -29,65 +29,8 @@
 
 #import "SFSDKLogger.h"
 
-@interface SFSDKAnalyticsLogger : NSObject
+extern NSString * _Nonnull const kSFSDKAnalyticsComponentName;
 
-/**
```

Comment:
Basically, all of the component-specific code can simply be pulled up into the parent class, leaving only the component configuration.